static/scripts/install-nim-bundle.sh

```diff
@@ -335,24 +337,30 @@ installBundleForDebianDistro() {
   debian_install_clickhouse
   debian_install_nim
   if [ "${USE_SM_MODULE}" == "true" ]; then
-      printf "Installing security module...\n"
-      if [ "${NIM_SM_VERSION}" == "latest" ]; then
-        apt-get install -y nms-sm
-        check_last_command_status "apt-get install -y nms-sm" $?
+      nim_major_version=$(nms-core --version | grep -oE '[0-9]+\.[0-9]+\.[0-9]+' | awk -F. '{print $1}')
+      nim_minor_version=$(nms-core --version | grep -oE '[0-9]+\.[0-9]+\.[0-9]+' | awk -F. '{print $1}')
+      if [[ $nim_major_version -ge 2 && $nim_minor_version -ge 19 ]]; then
+        echo "Warning: NIM version >=2.19.0 comes with security module installed already. skipping installing security module"
```

Comment:
This isn't really a warning, IMO

```suggestion
        echo "Note: NIM version >=2.19.0 comes with security module installed"
```

Reply:
Write out `>=2.19.0` in plain English instead of as symblols.

> NIM version 2.19.0 and later.

Reply:
I'm wondering if "module" is what we should be calling this now -- it's a feature, a dashboard that's part of NIM.

Reply:
Maybe with a modifier? (unsure)

"NGINX Instance Manager .... now with security monitoring installed"